apis/management.cattle.io/v3/multi_cluster_app.go

```diff
@@ -80,3 +81,8 @@ type MultiClusterAppRevision struct {
 type MultiClusterAppRollbackInput struct {
 	RevisionName string `json:"revisionName,omitempty" norman:"type=reference[multiClusterAppRevision]"`
 }
+
+type UpdateMultiClusterAppTargetsInput struct {
+	Projects []string `json:"projects" norman:"type=array[reference[project]],required"`
+	Answers  []Answer `json:"answers" norman:"type=array[reference[answer]]"`
+}
```

Comment:
Should not this accept []Target? We seem to miss the appName/Healthstate from Targets?

Reply:
App name is assigned by the controllers when apps are created for those targets, and so should healthstate eventually when we have logic for it